brave/browser-laptop
# Update Windows titlebar code to ignore AltGr

Update Windows titlebar code to ignore AltGr

- Implemented by adding new keyLocations file and checking event location.
- New file and existing keyCodes file were moved to new directory structure

Fixes https://github.com/brave/browser-laptop/issues/4626

Auditors: @bbondy 

## Test plan
1. Use Brave on Windows and visit http://fsymbols.com/keyboard/windows/alt-codes/list/
2. Ensure hide menu by default is enabled
3. Send focus to the demo text field, hold AltGr and then hit 1 to type a smiley face ☺
4. Notice menu is NOT toggled
